pkg/pillar/cmd/tpmmgr/tpmmgr.go

```diff
@@ -22,7 +22,7 @@ import (
 	"reflect"
 	"time"
 
-	"github.com/google/go-tpm/tpm2"
+	"github.com/google/go-tpm/legacy/tpm2"
```

Comment:
Tell me more about the legacy package and why we need to switch to it.

Reply:
go-tpm has moved the old tpm2 package (the one we have been using and is in pillar vendor dir) to "go-tpm/legacy/tpm2", now "go-tpm/tpm2" package contains a completely new and incompatible interface for interacting with TPM.

The reason for changing it that the package we have in vendor is old and doesn't have [credactivation](https://github.com/google/go-tpm/tree/main/legacy/tpm2/credactivation) package, so to use it I have to upgrade tpm2 package, and to continue to use the old interface, should use the "legacy/tpm2" instead of "tpm2".

Reply:
OK, thanks.